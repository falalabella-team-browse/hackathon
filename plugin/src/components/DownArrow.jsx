import React from "react";

const DownArrow = ({ fill = "#000", size = "15px", rotate = "0" }) => {
  return (
    <svg
      version="1.1"
      id="Layer_1"
      xmlns="http://www.w3.org/2000/svg"
      x="0px"
      y="0px"
      viewBox="0 0 512.011 512.011"
      height={size}
      width={size}
      style={{ transform: `rotate(${rotate}deg)` }}
    >
      <g>
        <g>
          <path
            fill={fill}
            d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0
			s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667
			C514.096,145.416,514.096,131.933,505.755,123.592z"
          />
        </g>
      </g>
    </svg>
  );
};

export default DownArrow;
