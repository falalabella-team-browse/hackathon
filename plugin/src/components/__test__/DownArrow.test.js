import React from "react";
import { shallow } from "enzyme";
import DownArrow from "../DownArrow";

describe("DownArrow", () => {
  it("should render properly, with default props", () => {
    const wrapper = shallow(<DownArrow />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should render properly, with props", () => {
    const wrapper = shallow(<DownArrow fill="#000" size="15px" rotate="0" />);
    expect(wrapper).toMatchSnapshot();
  });
});
