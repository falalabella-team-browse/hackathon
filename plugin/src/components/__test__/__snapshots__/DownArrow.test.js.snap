// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownArrow should render properly, with default props 1`] = `ShallowWrapper {}`;

exports[`DownArrow should render properly, with props 1`] = `ShallowWrapper {}`;
